### PR TITLE
Allow nil from boxing 

### LINKED ISSUE #54: No mapping if value from custom boxing is nil
If the value from a boxing method (e.g. in a custom category) returns `nil` no mapping on the instance happens and the old value persists. (see: https://github.com/krzysztofzablocki/KZPropertyMapper/blob/master/KZPropertyMapper/KZPropertyMapper.m#L164)

Is there any chance to set a property to nil for custom boxed value?

### PR DESCRIPTION
**Pull request by Miiha**
This PR addresses a bug where nil values from boxing resulted in an invalid mapping.

This probably fixes #54.

### CODE REVIEW

File: Example/ExampleTests/KZPropertyMapperTests.m [hunk 1]
```diff
@@ -93,6 +93,20 @@
           [KZPropertyMapper mapValuesFrom:sourceDictionary toInstance:testObject usingMapping:mapping];
           [[testObject.contentURL should] equal:[NSURL URLWithString:@"http://test.com/video.mp4"]];
         });
+
+        it(@"should handle nil from any boxing", ^{
+          sourceDictionary = @{@"videoURL" : [NSNull null]};
```
Comment (krzysztofzablocki) [inline Example/ExampleTests/KZPropertyMapperTests.m]: this isn't really using boxing though?

Comment (Miiha) [reply Example/ExampleTests/KZPropertyMapperTests.m]: The `videoURL`  property on the `TestObject` uses the buildin NSURL-boxing

Comment (krzysztofzablocki) [reply Example/ExampleTests/KZPropertyMapperTests.m]: ah right

**Comment by krzysztofzablocki** [comment]
thanks for contributing :)

**Pull request merged**